app/src/main/java/com/github/dedis/student20_pop/OrganizerActivity.java

```diff
@@ -55,6 +63,22 @@ public void onClick(View view) {
                 //Future: different Home UI for organizer (without connect UI?)
                 showFragment(new HomeFragment(), HomeFragment.TAG);
                 break;
+            case R.id.tab_identity:
+                Bundle bundle = new Bundle();
+
+                final PoPApplication app = ((PoPApplication) getApplication());
+                bundle.putString(PRIVATE_KEY_TAG, app.getPerson().getAuthentication());
+
+                //TODO : Retrieve this LAO from the Intent
+                Lao lao = new Lao("LAO I just joined", new Date(), new Keys().getPublicKey());
+                bundle.putString(LAO_ID_TAG, lao.getId());
+
+                // set Fragmentclass Arguments
+                IdentityFragment identityFragment = new IdentityFragment();
+                identityFragment.setArguments(bundle);
+                showFragment(identityFragment, IdentityFragment.TAG);
```

Comment:
Is the identity UI the same if we're attendee or organizer ?

Reply:
For the moment, in ordre to keep it simple, I assumed that it was thes cas. There is a comment about that suject In the specifications, it is mentionned the following : 
" The paradigm might be that you’re always anonymous if you’re an ordinary attendee and always non-anonymous if you’re an organizer or witness. "
But I think we can still change that in the future